### PR TITLE
Initial standout development

### PR DESCRIPTION
**Pull request by Matt-Butler**

### CODE REVIEW

File: src/index-template.ejs [hunk 1]
```diff
@@ -8,6 +8,11 @@
   </head>
   <body>
 
+    <h2>Standout</h2>
```
Comment (gneatgeek) [inline src/index-template.ejs]: Should be `<h1>` since it's the first heading in the document.

**Comment by Matt-Butler** [comment]
standout
standout
standout-notice
standout-notice

File: src/index-template.ejs (hunk 1 above)
Comment (Matt-Butler) [inline src/index-template.ejs]: updated

**Comment by gneatgeek** [comment]
👍

File: src/terra-standout.scss [hunk 2]
```diff
@@ -1,3 +1,25 @@
...
+  border-radius: 0.25em;
+  margin: 1em;
+  padding: 1.25em;
+
+  h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6 {
```
Comment (bjankord) [inline src/terra-standout.scss]: Change the `h1-h6` selectors to a class, `.terra-Standout-heading`. This reduces selectors and lets consumers opt-in to this styling.

File: src/terra-standout.scss [hunk 3]
```diff
@@ -1,3 +1,25 @@
 @import 'node_modules/terra-mixins/src/terra-mixins';
 @import './terra/standout/variables';
-@import './terra/standout/mixins';
+
+
+.terra-Standout {
+  background-color: $terra-standout-background-color;
+  border: 1px solid $terra-standout-border-color;
+  border-radius: 0.25em;
+  margin: 1em;
```
Comment (bjankord) [inline src/terra-standout.scss]: Does UX want margin all around the standout? I would guess they would want margin only on the bottom.

**Comment by Matt-Butler** [comment]
standout

After updating margins

File: src/terra-standout.scss (hunk 2 above)
Comment (Matt-Butler) [inline src/terra-standout.scss]: Updated. Thinking about this more, it isn't really the header than needs a margin-top of 0, it is the first element.

**Comment by bjankord** [comment]
+1

**Pull request merged**